hveto/const.py

```diff
@@ -0,0 +1,33 @@
+import os
+from gwpy.time import from_gps
+
+
+def get_hvetopath(gpstime):
+
+    """ Returns the path of hveto output files
+
+    Given a gpstime, the path of the folder
+    containing hveto trigger files is returned
+
+    Parameters
+    ----------
+    gpstime : `str` or `float`
+    start time of the day for this analysis
+
+    Returns
+    _______
+    path : `str`
```

Comment:
I'd say a short description of what this variable is couldn't hurt, something like:

```suggestion
    path : `str`
        path to the hveto output file on the local filesystem
```

Reply:
Done. Added the description.